### PR TITLE
Port System.Reflection new docs (second pass)

### PR DESCRIPTION
**Pull request by carlossanlop**
Missed some in the previous PR.

Contributes to https://github.com/dotnet/runtime/issues/105982

### CODE REVIEW

**Comment by carlossanlop** [comment]
The CI found a warning in an unrelated file, I just fixed it: System.Runtime.InteropServices/NFloat.xml

**Comment by carlossanlop** [comment]
Ping @gewarren, @dotnet/area-system-reflection and @dotnet/area-system-reflection-metadata. Can I please get a review?

File: xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml [hunk 1]
```diff
@@ -147,11 +148,13 @@
         <Parameter Name="mappedFieldData" Type="System.Reflection.Metadata.BlobBuilder" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="ilStream">To be added.</param>
-        <param name="mappedFieldData">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="ilStream">Outputs <see cref="T:System.Reflection.Metadata.BlobBuilder" /> bytes that includes all method's IL (body) emitted.</param>
+        <param name="mappedFieldData">Outputs <see cref="T:System.Reflection.Metadata.BlobBuilder" /> bytes that includes all field RVA data defined in the assembly.</param>
```
Comment (gewarren) [inline xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml:152]:
```suggestion
        <param name="mappedFieldData">When this method returns, contains the <see cref="T:System.Reflection.Metadata.BlobBuilder" /> bytes that includes all field RVA data defined in the assembly.</param>
```

File: xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml [hunk 2]
```diff
@@ -181,12 +184,14 @@
         <Parameter Name="pdbBuilder" Type="System.Reflection.Metadata.Ecma335.MetadataBuilder" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="ilStream">To be added.</param>
-        <param name="mappedFieldData">To be added.</param>
-        <param name="pdbBuilder">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="ilStream">Outputs <see cref="T:System.Reflection.Metadata.BlobBuilder" /> bytes that includes all method's IL (body) emitted.</param>
```
Comment (gewarren) [inline xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml:187]:
```suggestion
        <param name="ilStream">When this method returns, contains the <see cref="T:System.Reflection.Metadata.BlobBuilder" /> bytes that includes all method's IL (body) emitted.</param>
```

File: xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml [hunk 3]
```diff
@@ -181,12 +184,14 @@
...
+        <param name="mappedFieldData">Outputs <see cref="T:System.Reflection.Metadata.BlobBuilder" /> bytes that includes all field RVA data defined in the assembly.</param>
```
Comment (gewarren) [inline xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml:188]:
```suggestion
        <param name="mappedFieldData">When this method returns, contains the <see cref="T:System.Reflection.Metadata.BlobBuilder" /> bytes that includes all field RVA data defined in the assembly.</param>
```

File: xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml [hunk 4]
```diff
@@ -181,12 +184,14 @@
...
+        <param name="pdbBuilder">Outputs <see cref="T:System.Reflection.Metadata.Ecma335.MetadataBuilder" /> that includes PDB metadata.</param>
```
Comment (gewarren) [inline xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml:189]:
```suggestion
        <param name="pdbBuilder">When this method returns, contains the <see cref="T:System.Reflection.Metadata.Ecma335.MetadataBuilder" /> that includes PDB metadata.</param>
```

File: xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml [hunk 5]
```diff
@@ -306,9 +311,13 @@
         <Parameter Name="stream" Type="System.IO.Stream" />
       </Parameters>
       <Docs>
-        <param name="stream">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="stream">The <see cref="T:System.IO.Stream" /> to which the assembly serialized.</param>
```
Comment (gewarren) [inline xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml:314]:
```suggestion
        <param name="stream">The <see cref="T:System.IO.Stream" /> to which the assembly should be serialized.</param>
```

File: xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml [hunk 6]
```diff
@@ -306,9 +311,13 @@
...
+        <summary>Serializes the assembly to <see cref="T:System.IO.Stream" />.</summary>
```
Comment (gewarren) [inline xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml:315]:
```suggestion
        <summary>Serializes the assembly to the specified stream.</summary>
```

File: xml/System.Reflection.Emit/ILGenerator.xml [hunk 7]
```diff
@@ -517,9 +517,9 @@
         <Parameter Name="id" Type="System.Int32" Index="0" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <param name="id">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="id">The unique id for the label.</param>
```
Comment (gewarren) [inline xml/System.Reflection.Emit/ILGenerator.xml:520]:
```suggestion
        <param name="id">The unique ID for the label.</param>
```

File: xml/System.Reflection.Emit/ILGenerator.xml [hunk 8]
```diff
@@ -517,9 +517,9 @@
...
+        <summary>Creates a <see cref="T:System.Reflection.Emit.Label" /> with the given id.</summary>
```
Comment (gewarren) [inline xml/System.Reflection.Emit/ILGenerator.xml:521]:
```suggestion
        <summary>Creates a <see cref="T:System.Reflection.Emit.Label" /> with the given ID.</summary>
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**